DuckDuckGo/HomeViewController.swift

```diff
@@ -110,6 +113,10 @@ class HomeViewController: UIViewController {
         infoViewHeight.constant = 0
     }
     
+    private func applyHomeRowCTA(type: HomeRowCTA.CTAType) {
```

Comment:
**opinion, not a request**

maybe a personal style thing, but i wouldn't have stub methods like this in the code. for one, "experiment 2" implies a lot of hidden knowledge that isn't obvious from the code.

when experiment 2 starts, i'd add the function in but not before - because there's always the chance that "experiment 2" will never start and you'd be the only person confident to remove this function/comment.

Reply:
Generally agree, but we're definitely doing experiment 2 :)